services/frontend-service/src/ui/App/index.tsx

```diff
@@ -21,38 +21,59 @@ import '../../assets/app-v2.scss';
 import * as React from 'react';
 import { PanicOverview, UpdateOverview } from '../utils/store';
 import { useApi } from '../utils/GrpcApi';
+import { AzureAuthProvider, UpdateFrontendConfig, useAzureAuthSub } from '../utils/AzureAuthProvider';
 
 export const App: React.FC = () => {
     const api = useApi;
+    const { authHeader, authReady } = useAzureAuthSub((auth) => auth);
 
     React.useEffect(() => {
-        const subscription = api
-            .overviewService()
-            .StreamOverview({}) // TODO TE: add auth header
-            .subscribe(
+        api.configService()
+            .GetConfig({})
+            .then(
                 (result) => {
-                    UpdateOverview.set(result);
-                    PanicOverview.set({ error: '' });
+                    UpdateFrontendConfig.set({ configs: result, configReady: true });
                 },
-                (error) => PanicOverview.set({ error: JSON.stringify(error) })
+                (error) => {
+                    // eslint-disable-next-line no-console
+                    alert('Error: Cannot connect to server!\n' + error);
```

Comment:
can't we use the normal way to display errors? An `alert` is quite intrusive

Reply:
Added a [story](https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/-M37cZx7XYoOvQ5Mmcir/STORY/-NEkhGYEXWbhiSAJ216v/)